service/contractsv1/api.go

```diff
@@ -0,0 +1,194 @@
+package contractsv1
+
+import "fmt"
+
+func depthAllowedValues(val string) (ok bool) {
+	values := []string{"TOP", "ALL"}
+	for i := range values {
+		if ok = values[i] == val; ok {
+			return
+		}
+	}
+	return
+}
+
+// ListContracts gets the list of contracts that a user has access to.
+// 'depth' returns a specific set of contracts.
+// Select TOP to return only parent contracts or ALL to return both parent and child contracts.
+func (c *Contractsv1) ListContracts(depth string) (*OutputContractIDs, error) {
```

Comment:
Cannot this be done in the same way we have network lists ? Or with constants specific types ?

Reply:
Done